.travis.yml

```diff
@@ -1,46 +0,0 @@
-sudo: required
-dist: trusty
-services:
-- docker
-env:
-  global:
-  - COMMIT_AUTHOR_EMAIL=skynet@open.qa
-matrix:
-  include:
-    - name: "unit- and integration tests"
-      env: FULLSTACK=0 UITESTS=0
-    - name: "UI tests"
-      env: FULLSTACK=0 UITESTS=1
-    - name: "fullstack test"
-      env: FULLSTACK=1
-    - name: "scheduler fullstack test"
-      env: SCHEDULER_FULLSTACK=1
-    - name: "developer fullstack test"
-      env: DEVELOPER_FULLSTACK=1
-    - name: "documentation generation job"
-      env: BUILD_DOCS=1
-before_script:
-- docker pull registry.opensuse.org/devel/openqa/containers/openqa_dev:latest
-cache:
-  directories:
-  - assets/cache
-script:
-  - |
-    if [ -n "$BUILD_DOCS" ]; then
-      bash script/generate-documentation $encrypted_e2c381aa6b8c_key $encrypted_e2c381aa6b8c_iv
-    else
-      mkdir -p assets/cache && chmod a+w assets/cache && sudo chown -R 1000:1000 assets/cache
-      touch openqa-debug.log autoinst-log.txt && chmod a+w openqa-debug.log autoinst-log.txt
-      # as long as we have some unfixed flaky tests we should let travis CI retry,
-      # not humans :)
-      # known issues:
-      # * 05-scheduler-full.t: https://progress.opensuse.org/issues/37628
-      # * 25-cache-service.t: https://progress.opensuse.org/issues/53858
-      # * ui/12-needle-edit.t: https://progress.opensuse.org/issues/37610
-      # * ui/18-tests-details.t: https://progress.opensuse.org/issues/38276
-      # * full-stack.t: https://progress.opensuse.org/issues/37638 and https://progress.opensuse.org/issues/54032
-      travis_retry make prepare-and-launch-docker-to-run-tests-within
```

Comment:
Did we manage to stabilize all tests yet? Because I am not aware of this. If that changed, could you please also provide an update in the tickets?

Reply:
The PR introduces `make test-unstable` command which re-tries every test from `t/unstable-tests.txt` several times